### DISCUSSION TITLE
How would someone upgrade the self hosted docmost version

### DISCUSSION THREAD

**Discussion by prathamesh-0909** [docmost/docmost#248] [Help / Questions]
We are using Docmost as our primary wiki, knowledge base. We are using the self hosted version one, curious to ask how are you planning to provide upcoming upgrades in the further versions of docmost.

Thank you

**Comment by Philipinho** [comment]
I assume you are using docker compose.

```sh
docker pull docmost/docmost:latest
docker compose up --force-recreate --build docmost -d
```
https://docmost.com/docs/installation/#upgrade
